samples/ClassroomLabs/Modules/Library/Tools/LabCreationLibrary.psm1

```diff
@@ -338,29 +338,55 @@ function Publish-Labs {
                 $ConfigObject
             )
 
-            $lacs = $ConfigObject | Select-Object -Property ResourceGroupName, LabAccountName -Unique
+            $lacs = $ConfigObject | Select-Object -Property ResourceGroupName, LabAccountName, SharedGalleryResourceGroupName, SharedGalleryName -Unique
```

Comment:
I'm not sure we should do it this way...  This means that the bulk lab creation scripts can only use a shared image gallery from the current subscription.  We should either add a "SharedGallerySubscriptionId" or we should consider just having a single field with the full shared image gallery resource ID.  (I like the single field idea, because it could just be an empty column or missing entirely if the user doesn't need it - vs needing to do validation across 3 columns to make sure there isn't a missing field...